docs/src/pages/guides/auth-protected/auth-protected.angular.mdx

```diff
@@ -0,0 +1 @@
+## Coming Soon
```

Comment:
For now, let's focus on React, and I'll add guides to Angular and Vue in the future.